builder/site.hs

```diff
@@ -38,29 +67,34 @@ main = mkContext >>= \ctx -> hakyll $ do
     compile templateCompiler
 
 
+parseTestimonialCompiler :: Compiler (Item Testimonial)
+parseTestimonialCompiler = do
+  identifier  <- getUnderlying
+  provider    <- compilerProvider <$> compilerAsk
```

Comment:
Internal import is used here. Can these be found in a non-internal module?

Reply:
If we update to the latest hakyll this is part of the public API. I'm happy to do it, but I wonder if maybe we'd be better off merging this as-is and then taking on the update as a second piece of work afterwards? I'd be happy to take on the update as well.

Reply:
I'll leave the decision up to you. Personally I'm happy to merge now and update later.